index.js

```diff
@@ -97,12 +123,21 @@ console.log('hello world')
  * 1. Create a function called `stringToUpper`
  * 2. Add a parameter called `str` in your function signature
  * 3. Add a return statement in your function which returns the input string in all uppercase letters
- * 4. Use `console.log` to print out the output of `stringToUpper(str)`
+ * 4. Use `console.log` to print out the output of `stringToUpper(s
  *
  * Example Output: "WHY AM I SCREAMING"
  */
 
-// TODO: ADD CODE HERE
+//  Function Declaration:
+// function stringToUpper(str) {
+//     return str.toUpperCase();
+// }
+// console.log(stringToUpper("I'm almost done with my homework"));
+
+// Arrow Function:
+const stringToUpper = (str) => str.toUpperCase();
```

Comment:
nice use of returning directly in the arrow function!
When there is only one parameter in an arrow function, the parentheses are optional, so you could also write this as
`const stringToUpper = str => str.toUpperCase();`

Reply:
@marymatt504 , thank you so much for reviewing my code